packages/client/integration/Loc.ts

```diff
@@ -229,28 +238,42 @@ export async function collectionLocWithUpload(state: State) {
         description: "This is the Logion Classification LOC",
         draft: false,
     });
-    locsState = logionClassificationLocRequest.locsState();
-    await legalOfficer.openAndClose(logionClassificationLocRequest.locId);
+    await legalOfficer.acceptLoc(logionClassificationLocRequest.locId);
+
+    const logionClassificationAcceptedRequest = await logionClassificationLocRequest.refresh() as AcceptedRequest;
+    const logionClassificationOpenLoc = await logionClassificationAcceptedRequest.open({ signer });
+
+    await legalOfficer.closeLoc(logionClassificationAcceptedRequest.locId);
+
+    locsState = logionClassificationOpenLoc.locsState();
     const creativeCommonsLocRequest = await locsState.requestTransactionLoc({
         legalOfficer: client.getLegalOfficer(alice.address),
         description: "This is the LOC acting usage of CreativeCommons on logion",
         draft: false,
     });
-    locsState = creativeCommonsLocRequest.locsState();
-    await legalOfficer.openAndClose(creativeCommonsLocRequest.locId);
+    await legalOfficer.acceptLoc(creativeCommonsLocRequest.locId);
+
+    const creativeCommonsAcceptedRequest = await creativeCommonsLocRequest.refresh() as AcceptedRequest;
+    const creativeCommonsOpenLoc = await creativeCommonsAcceptedRequest.open({ signer });
 
+    await legalOfficer.closeLoc(creativeCommonsAcceptedRequest.locId);
```

Comment:
Replace by a call to new LO API?

Reply:
Good suggestion. This will be addressed later on in logion-network/logion-internal#923